dbinspector/build.gradle

```diff
@@ -32,59 +32,59 @@ dependencies {
     compile 'com.android.support:appcompat-v7:21.0.3'
 }
 
-apply plugin: 'maven'
-apply plugin: 'signing'
-
-version = '2.0.0'
-group = 'im.dino'
-
-configurations {
-    archives {
-        extendsFrom configurations.default
-    }
-}
-
-signing {
-    required { has("release") && gradle.taskGraph.hasTask("uploadArchives") }
-    sign configurations.archives
-}
-
-uploadArchives {
-    configuration = configurations.archives
-    repositories.mavenDeployer {
-        beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
-
-        repository(url: sonatypeRepo) {
-            authentication(userName: sonatypeUsername,
-                    password: sonatypePassword)
-        }
-
-        pom.project {
-            name 'Android DbInspector'
-            packaging 'aar'
-            description 'Android library for viewing in app databases.'
-            url 'https://github.com/infinum/android_dbinspector/'
-
-            scm {
-                url 'scm:git@github.com:infinum/android_dbinspector.git'
-                connection 'scm:git@github.com:infinum/android_dbinspector.git'
-                developerConnection 'scm:git@github.com:infinum/android_dbinspector.git'
-            }
-
-            licenses {
-                license {
-                    name 'The Apache Software License, Version 2.0'
-                    url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
-                    distribution 'repo'
-                }
-            }
-
-            developers {
-                developer {
-                    id 'reisub'
-                    name 'Dino Kovač'
-                }
-            }
-        }
-    }
-}
+//apply plugin: 'maven'
```

Comment:
@PrashamTrivedi Please don't change/comment the deploy code, we need it to deploy DbInspector to maven central.